src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -439,6 +439,17 @@ public static bool IsTargetReadyForAeWithKeyStore()
                 ;
         }
 
+        public static bool IsSupportingDistributedTransactions()
```

Comment:
Decided to use fully qualified names here to avoid having to do `if/def` in the using section